# /controller/srv/AutoInitPos.srv
string caller
---
bool success # 结果
